app/components/provider_interface/conditions_list_component.rb

```diff
@@ -0,0 +1,9 @@
+module ProviderInterface
+  class ConditionsListComponent < ViewComponent::Base
+    attr_reader :conditions
+
+    def initialize(conditions)
+      @conditions = conditions || []
```

Comment:
The association should always return an empty array if it's not set so I don't think this is needed.

```suggestion
      @conditions = conditions 
```